roles/set-external-eth1/tasks/main.yaml

```diff
@@ -21,9 +21,9 @@
 # LODESTAR
 - name: Set external eth1 for lodestar
   lineinfile:
-    path: "{{ e2dc_install_path }}/config/lodestar/beacon.json"
-    regexp: '^\s*"providerUrl":.*'
-    line: '        "providerUrl": "{{ connectivity.eth1_nodes.0 | quote }}"'
+    path: "{{ e2dc_install_path }}/config/lodestar/beacon.yaml"
+    regexp: '^eth1.providerUrls:.*'
+    line: 'eth1.providerUrls: {{ connectivity.eth1_nodes.0 }}'
```

Comment:
this should support multiple urls, maybe do it like this: https://github.com/stereum-dev/ethereum2-ansible/blob/7f8acfe957b0c510be2ef47d46bc19f3087548ca/roles/set-external-eth1/tasks/main.yaml#L75